load.go

```diff
@@ -142,11 +147,11 @@ func (l *Loader) LoadURL(url string) (*Properties, error) {
 		return nil, fmt.Errorf("properties: invalid content type %s", ct)
 	}
 
-	return l.loadBytes(body, enc)
+	return l.loadBytes(body, enc, l.PreserveFormatting)
 }
 
-func (l *Loader) loadBytes(buf []byte, enc Encoding) (*Properties, error) {
-	p, err := parse(convert(buf, enc))
+func (l *Loader) loadBytes(buf []byte, enc Encoding, preserveFormatting bool) (*Properties, error) {
```

Comment:
Since all `loadXXX` functions eventually call `loadBytes` you don't need pass the parameter through. you can just use `l.PreserveFormatting`

Reply:
Removed the passed parameters in load.go.